tests/nrf5340_audio/sd_card/src/main.c

```diff
@@ -0,0 +1,283 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/ztest.h>
+#include <zephyr/device.h>
+#include <zephyr/storage/disk_access.h>
+#include <modules/sd_card.h>
+#include <zephyr/fs/fs.h>
+#include <ff.h>
+#include <math.h>
+
+#define MKFS_DEV_ID "SD:"
+
+static int test_create_file(const char *path)
+{
+	int ret;
+	struct fs_file_t file;
+
+	fs_file_t_init(&file);
+	ret = fs_open(&file, path, FS_O_CREATE | FS_O_WRITE);
+	if (ret == 0) {
+		fs_write(&file, "Testfile", 8);
+		fs_close(&file);
+	} else {
+		TC_PRINT("Failed to open file\n");
+		return ret;
+	}
+
+	return 0;
+}
+
+static int dir_name_get(char *name, size_t name_size, uint32_t curr_level, uint32_t curr_dir_id,
+			uint8_t subdirs)
+{
+	int ret;
+
+	name[0] = '\0';
+	strcat(name, "/SD:");
+
+	struct dir_tree {
+		uint32_t peer_id;
+		uint32_t name;
```

Comment:
`name` is a uint32_t? I guess in this case the level is sort of the name, but for clarity I recommend a rename

Reply:
Changed. This is the numeric part of the name, but I can understand that this is hard to read.